docs/pages/versions/unversioned/sdk/sqlite-next.mdx

```diff
@@ -343,6 +343,23 @@ const row = await db.getFirstAsync<{ data: Uint8Array }>('SELECT * FROM blobs');
 expect(row.data).toEqual(blob);
 ```
 
+## Third-party library integrations
+
+The `expo-sqlite` library is designed to be a solid SQLite foundation. It enables broader integrations with third-party libraries for more advanced higher-level features. Here are some of the libraries that you can use with `expo-sqlite`.
+
+### Drizzle ORM
+
+[Drizzle](https://orm.drizzle.team/) is a ["headless TypeScript ORM with a head"](https://orm.drizzle.team/docs/overview). It runs on Node, Bun, Deno, and React Native. It also comes with a [CLI compansion called drizzle-kit](https://orm.drizzle.team/kit-docs/overview) for generating SQL migrations.
```

Comment:
```suggestion
[Drizzle](https://orm.drizzle.team/) is a ["headless TypeScript ORM with a head"](https://orm.drizzle.team/docs/overview). It runs on Node.js, Bun, Deno, and React Native. It also has a CLI compansion called [`drizzle-kit`](https://orm.drizzle.team/kit-docs/overview) for generating SQL migrations.
```

Let's highlight the CLI tool/package name with inline code highlight and for link, only highlight the library name as well.